### PR TITLE
Add some docs on creating a GKE cluster managed by terraform

### PR DESCRIPTION
**Pull request by sgibson91**
Includes the extra steps I currently have to do for the Pangeo project.

### CODE REVIEW

File: docs/howto/operate/new-tf-cluster.md
Comment (yuvipanda) [inline docs/howto/operate/new-tf-cluster.md:36]: For terraform, only the `gcloud auth application-default login` is needed I think.

File: docs/howto/operate/new-tf-cluster.md
Comment (yuvipanda) [inline docs/howto/operate/new-tf-cluster.md:68]:
```suggestion
Create a new workspace with the below command, and again give it the same name as the `.tfvars` filename.
```

**Review by yuvipanda — APPROVED** [review]
Minor suggestion but happy to merge after that!

File: docs/howto/operate/new-tf-cluster.md
Comment (sgibson91) [inline docs/howto/operate/new-tf-cluster.md:35]:
```suggestion
```

File: docs/howto/operate/new-tf-cluster.md
Comment (sgibson91) [inline docs/howto/operate/new-tf-cluster.md:54]:
```suggestion
gcloud auth application-default print-access-token
```

Fix typo

**Comment by yuvipanda** [comment]
Thanks, @sgibson91!

File: docs/howto/operate/new-tf-cluster.md
Comment (damianavila) [inline docs/howto/operate/new-tf-cluster.md:3]: guide

File: docs/howto/operate/new-tf-cluster.md
Comment (damianavila) [inline docs/howto/operate/new-tf-cluster.md:58]: Wondering how we can prevent this from happening... I guess since this is a rare scenario we should just pay double attention to it... but it is something easy to miss, IMHO.

File: docs/howto/operate/new-tf-cluster.md
Comment (damianavila) [inline docs/howto/operate/new-tf-cluster.md:86]: I think that should be a requirement to fulfill before merging the PR... which actually led me to think this guide is not talking about the associated post-deployment PR, is that intended (to be added later)?

**Comment by damianavila** [comment]
Belated comments @sgibson91, sorry about that, hopefully, some of them are useful.

File: docs/howto/operate/new-tf-cluster.md
Comment (sgibson91) [reply docs/howto/operate/new-tf-cluster.md:86]: I'm not sure, how does the post-deployment PR work?

Comment (damianavila) [reply docs/howto/operate/new-tf-cluster.md:86]: I think I was not clear enough... the process you outlined here "generates" a `.tfvars` file at a minimum. That one needs to live in a PR that eventually will be merged. I think the terraform plan output should be posted in that PR and revisited by a pair before actually deploying the cluster (maybe post-PR is not the best name). Even when there are mentions about others checking the plan, there is not an "associated" PR with the cluster generation event. 
So, I am trying to codify in the document how the process looked like for your work with the pangeo cluster and I am trying to follow the model we outlined for the org-ops in this section: https://pilot-hubs.2i2c.org/en/latest/howto/operate/setup-new-project.html#client-organization-provides-billing-account, maybe that helps with additional context.

Comment (sgibson91) [reply docs/howto/operate/new-tf-cluster.md:86]: Ah, gotcha! Thanks for clarifying :) I'll open up another PR and try to make those two match

**Pull request merged**
